rpmlint/configdefaults.toml

```diff
@@ -29,6 +29,8 @@ Checks = [
 ]
 # Various output filters, list of regexp strings eg. "E: .* no-signature"
 Filters = []
+# List of errors that can't be filtered
+FilterExceptions = []
```

Comment:
I don't think this is an intuitive name for this. The idea here is that check rules listed here must pass, right?

Reply:
Hm, you are right. What about `SeriousErrors`, `SecurityErrors` or `FatalErrors`. Or do you have a more suitable name?

Reply:
`FatalErrors` sounds better. This could also turn a warning into an error, right?

Reply:
it is not supposed to change the warning level at all. it is about suppressing filtering. 

we could also call it "FilterDenylist" or the like

Reply:
`BlockedFilters` maybe?